src/novatel_oem7_driver/config/std_msg_topics.yaml

```diff
@@ -1,5 +1,6 @@
 # ROS-standard
 IMU:        {topic: /gps/imu,                 frame_id: imu_link}
```

Comment:
Topics should never have a prepending slash, this prevents namespacing of the node (in case of multiple gps units or multiple robots for example).

Reply:
Thank you for your input. We have fixed this issue in our 'ROS2' branch, we will work on making the changes to 'ROS1' and 'Master' branch.

Reply:
Hi @Timple, we have had some discussions internally regarding making this change to ROS1. The biggest concern we have is that it will break backwards compatibility for a lot of existing users of our ROS1 branch driver.

This issue is fixed in our ROS2 branch, and we will have a hard switch to ROS2 at some point in the future, please see announcement: https://github.com/novatel/novatel_oem7_driver/discussions/34

To have the least amount of impact for existing users, would you consider making this change on your local build?

Reply:
Thank you for looking into this!

I don't thing it will break for a lot of existing users. Because if they where namespacing, it wouldn't work. And if they are not, it doesn't break :).

But we're using our own yaml file because of this, so feel free to leave it like this. Thank you for fixing ROS2